pydbg.py

```diff
@@ -1,10 +1,17 @@
 """pydbg is an implementation of the Rust2018 builtin `dbg` for Python."""
+
 import inspect
+import sys
+import typing
+
 
 __version__ = "0.2.1"
```

Comment:
Can you bump the version to `"0.3.0"` since it is a change from stdout to stderr?

Reply:
Sure. Will do!